### ISSUE TITLE
gitian build fails with checksum mismatch

### ISSUE THREAD

**Issue by MarcoFalke** [bitcoin/bitcoin#17315]
```
Starting gitian build ...
5d1962100da67257da5fc00b29825c0ccae0cfd6
HEAD is now at 5d1962100d Merge a932d1328333a20eee7c306e6fd681641777e95d into 6a97e8a060f7632bbaee27d3de8035dc6ebe3895

Building Dependencies

--2019-10-29 13:47:08--  https://downloads.sourceforge.net/project/osslsigncode/osslsigncode/osslsigncode-1.7.1.tar.gz
Resolving downloads.sourceforge.net (downloads.sourceforge.net)... 99.84.216.49, 99.84.216.120, 99.84.216.52, ...
Connecting to downloads.sourceforge.net (downloads.sourceforge.net)|99.84.216.49|:443... connected.
HTTP request sent, awaiting response... 302 Moved Temporarily
Location: https://downloads.sourceforge.net/#!/project/osslsigncode/osslsigncode/osslsigncode-1.7.1.tar.gz [following]
--2019-10-29 13:47:08--  https://downloads.sourceforge.net/
Reusing existing connection to downloads.sourceforge.net:443.
HTTP request sent, awaiting response... 200 OK
Length: 746 [text/html]
Saving to: ‘inputs/osslsigncode-1.7.1.tar.gz’

osslsigncode-1.7.1.tar.gz                                   100%[=========================================================================================================================================>]     746  --.-KB/s    in 0s      

2019-10-29 13:47:08 (9.29 MB/s) - ‘inputs/osslsigncode-1.7.1.tar.gz’ saved [746/746]

--2019-10-29 13:47:08--  https://bitcoincore.org/cfields/osslsigncode-Backports-to-1.7.1.patch
Resolving bitcoincore.org (bitcoincore.org)... 107.191.99.5, 198.251.83.116
Connecting to bitcoincore.org (bitcoincore.org)|107.191.99.5|:443... connected.
HTTP request sent, awaiting response... 304 Not Modified
File ‘inputs/osslsigncode-Backports-to-1.7.1.patch’ not modified on server. Omitting download.

inputs/osslsigncode-Backports-to-1.7.1.patch: OK
inputs/osslsigncode-1.7.1.tar.gz: FAILED
sha256sum: WARNING: 1 computed checksum did NOT match

subprocess.CalledProcessError: Command '["echo 'f9a8cdb38b9c309326764ebc937cba1523a3a751a7ab05df3ecc99d18ae466c9 inputs/osslsigncode-1.7.1.tar.gz' | sha256sum -c"]' returned non-zero exit status 1.
```

```
$ sha256sum scratch/gitian/gitian-builder/inputs/osslsigncode-1.7.1.tar.gz 
251a85b3bac687974f360d3796048c20ded3bf0bd69e0d1cfd1db23d013f89ed  scratch/gitian/gitian-builder/inputs/osslsigncode-1.7.1.tar.gz

**Comment by MarcoFalke** [comment]
[osslsigncode-1.7.1.tar.gz](https://github.com/bitcoin/bitcoin/files/3788909/osslsigncode-1.7.1.tar.gz)


```
$ cat scratch/gitian/gitian-builder/inputs/osslsigncode-1.7.1.tar.gz 
<html><head>
<title>SourceForge</title>
<!-- <script src="/js/jquery.com/jquery-1.11.0.min.js"></script> -->
<script src="https://code.jquery.com/jquery-1.11.0.min.js"></script>
<script src="https://sourceforge.net/js/mirrors.js"></script>
<script src="/js/sf.js"></script>
<script>
var DR_loc = DR_parse_hash_url();
if (DR_loc) {
    DR_sf_main(DR_loc);
} else {
    window.location.href = 'https://sourceforge.net/home.html';
}
</script>
<meta name="description" content="Free, secure and fast downloads from the largest Open Source applications and software directory - SourceForge.net"/>
</head><body>
<noscript>
We're sorry -- the Sourceforge site is currently in Disaster Recovery mode.  Please check back later.
</noscript>
</body></html>

**Comment by fanquake** [comment]
> Length: 746 [text/html]

I'd assume you've downloaded a html page, and not the actual osssigncode tar. I've just downloaded via the same link I get the correct shasum:

```bash
shasum -a 256 /Users/michael/Downloads/osslsigncode-1.7.1.tar.gz 
f9a8cdb38b9c309326764ebc937cba1523a3a751a7ab05df3ecc99d18ae466c9  /Users/michael/Downloads/osslsigncode-1.7.1.tar.gz
```

**Comment by MarcoFalke** [comment]
Yeah. "Disaster Recovery mode". What a useless website. Can't we get rid of it?

**Comment by laanwj** [comment]
Would be fixed by #16669 I guess?

**Comment by MarcoFalke** [comment]
Jup, it would be fixed by that.


Btw thanks to @TheCharlatan for adding the checksum check in 03b8596dd665d2f70c917794295911adb8680bcc

**Comment by laanwj** [comment]
Closed by #16669
